package.json

```diff
@@ -35,19 +35,19 @@
   },
   "peerDependencies": {
     "@appuniversum/ember-appuniversum": "^1.0.2",
-    "@lblod/ember-rdfa-editor": "^0.53.1",
+    "@lblod/ember-rdfa-editor": ">=0.55.0",
```

Comment:
kind of odd to require a less current version here than the one used in the dev dependencies, no?

Reply:
Maybe. This plugins works on a wide range of editor versions, since it does not use the really new stuff, but while testing, I want to make sure it is not broken on the latest version of the editor.